AzureWebFarm.OctopusDeploy/Infrastructure/OctopusDeploy.cs

```diff
@@ -106,6 +131,11 @@ public static IOctopusRepository GetRepository(ConfigSettings config)
 
         public void UninstallTentacle()
         {
+            var machineResource = _repository.Machines.FindByName(_machineName);

```

Comment:
Interesting. Mind raising this part of the change in a separate PR given its a separate change (and I'd like to understand why this wasn't already happening).



> On 24 Apr 2015, at 6:26 pm, Lee Baker <notifications@github.com> wrote:
> 
> In AzureWebFarm.OctopusDeploy/Infrastructure/OctopusDeploy.cs:
> 
> > @@ -106,6 +131,11 @@ public static IOctopusRepository GetRepository(ConfigSettings config)
> >  
> >          public void UninstallTentacle()
> >          {
> > +            var machineResource = _repository.Machines.FindByName(_machineName);
> I found that just uninstalling the tentacle didn't remove the machine from the ocotpus server. It just gave me an unresponsive machine that was holding up deployments. This makes sure it is deleted from the server.
> 
> —
> Reply to this email directly or view it on GitHub.
>